src/ffi.rs

```diff
@@ -231,6 +234,26 @@ pub extern fn quiche_config_set_cc_algorithm(
     config.set_cc_algorithm(algo);
 }
 
+#[allow(unused_variables)]
+#[no_mangle]
+#[cfg(all(unix, feature = "qlog"))]
+pub extern fn quiche_conn_set_qlog_fd(conn: &mut Connection, fd: c_int) -> c_int {
+    let f = unsafe { std::fs::File::from_raw_fd(fd) };
+    let writer = Some(std::io::BufWriter::new(f));
```

Comment:
Why is this wrapped in an `Option`? The `if` condition below will always be true.

Reply:
LOL. With applying your suggestions there is little that can co wrong with the method. I removed the return codes as part of simplifying